### DISCUSSION TITLE
[BUG] kamp enable does not create new mesh on print start

### DISCUSSION THREAD

**Discussion by gngrvkng** [bassamanator/Sovol-SV06-firmware#124] [General]
**Describe the bug**
after enabling kamp in misc-macros and including [exclude object] in printer config, at print start it does not perform bed mesh

**What printer are you using?**

- SV06 with sv06 plus extruder, same behavior before swapping extruder

**What kind of device are you using to host Klipper?**
raspberry pi 3b, mainsail install from pi imager, all updated

**Is your local copy of this configuration up to date?**
yes

- This configuration changes all the time, when did you install it? Are you sure you're running the latest version?
5 hrs ago

**What slicer are you using?**

sovol cura

**Are you using the config bundles from this repo?**

yes

**Did you [Adjust your slicer](https://github.com/bassamanator/Sovol-SV06-firmware#adjust-your-slicer)**?

yes, i removed original code and left blank in sovol cura

**Comment by boogietrain** [comment]
I think you have to use the PRINT_START macro to get.  Put 

PRINT_START

 as your start gcode in Cura and see if that works.

**Reply by gngrvkng** [reply]
Ok, will do. I'll give it a shot tonight. Thanks!

**Reply by gngrvkng** [reply]
sorry it, gives undfined code error, i'll try a new macro in cura. I'll copy paste your print_start config and try tomorrow

**Reply by gngrvkng** [reply]
unfortunately still no luck, i'll see what i can find online, thanks

**Comment by bassamanator** [comment]
I want you to read the `adjust your slicer` section again, but more carefully this time.

**Reply by gngrvkng** [reply]
I ended up reinstalling klipper again and this time it seemed to work flawlessly, thanks again for your help!

**Reply by bassamanator** [reply]
Great! Please star the repo.

**Reply by gngrvkng** [reply]
no problem, i thought i already did
